### ISSUE TITLE
No support for long project names with GitLab integration?

### ISSUE THREAD

**Issue by legovaer** [platformsh/platformsh-cli#708]
I'm trying to add an integration with GitLab and our group is managed in a way so that we have a clear understanding of what the repo is related to. It looks something like this:

```
(company)slug
  teamnameA
    clientX
      projectX
      projectY
    clientY
   
  teamnameB
    toolX
    toolY
    clientX
  teamname C
```

Now when I try to add an integration to platform for Project X for client X in team A, I use the following command:

`platform integration:add --type=gitlab --token=[MY-GITLAB-TOKEN] --base-url=https://gitlab.com/ --gitlab-project=slug/teamA/clientX/projectX` 

The CLI now throws an `InvalidValueException`:

> Invalid value for 'Project': slug/teamA/clientX/projectX

Is there a work around for this?

**Comment by pjcdawkins** [comment]
Fixed in version [3.32.2](https://github.com/platformsh/platformsh-cli/releases/tag/v3.32.2)

**Comment by legovaer** [comment]
Wow. That's quick 👍 

Thanks a lot!
